### PR TITLE
Changes to _config.yml; added image.

### PR DESCRIPTION
**Pull request by utksi**
Added a gif to the portfolio website.

### CODE REVIEW

**Comment by rartino** [comment]
Thanks for the PR, great addition!

File: _config.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 theme: jekyll-theme-minimal
 title: Test Testsson
 description: This is my portfolio website.
+logo: https://media.giphy.com/media/uQHtUvva9Qljy/giphy.gif
```
Comment (rartino) [inline _config.yml]: Perhaps another image would be better :-)

**Review by rartino — CHANGES_REQUESTED** [review]
Great, but some suggestions.

File: _config.yml (hunk 1 above)
Comment (utksi) [reply _config.yml]: The new one is better!

**Review by rartino — APPROVED** [review]
Now it look great!

**Pull request merged**